frontend/yarn.lock

```diff
@@ -0,0 +1,4240 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
I know nothing about yarn. Do we need this file to build the frontend now? I've been using `gulp`, but has this now changed?

Reply:
Nope, nothing we need to change. yarn is just an alternative to npm I have been using.
https://yarnpkg.com

Reply:
fab :)